File: subroutines/all/subroutine askstop.f
      subroutine askstop(idef)
      common /logging/ logfile,ipredict
      if (ipredict .eq. 0) then
        call askyn('Do you want to continue',23,1,idef,icont,20,0)
        if (icont .eq. 0) stop
      else
        print *,'Run continues as predictable input was requested'
      end if
      return
      end

File: fortran to python conversion/dependencies/subroutine askstop.py
def askstop(idef):
    """
    ***
    Parameters:
    idef:
    """
    if (ipredict == 0):
        askyn('Do you want to continue', 23, 1, idef, icont, 20, 0)
        if (icont == 0):
            break
    else:
        print("Run continues as predictable input was requested")
    return
